test/Microsoft.AspNetCore.Mvc.Razor.Test/Compilation/ViewsFeatureProviderTest.cs

```diff
@@ -36,7 +37,29 @@ public void PopulateFeature_ReturnsViewsFromAllAvailableApplicationParts()
             // Arrange
             var part1 = new AssemblyPart(typeof(object).GetTypeInfo().Assembly);
             var part2 = new AssemblyPart(GetType().GetTypeInfo().Assembly);
-            var featureProvider = new TestableViewsFeatureProvider(new Dictionary<AssemblyPart, IEnumerable<RazorViewAttribute>>
+
+            var items = new Dictionary<AssemblyPart, IReadOnlyList<RazorCompiledItem>>
+            {
+                {
+                    part1,
+                    new[]
+                    {
+                        new TestRazorCompiledItem(typeof(object), "mvc.1.0.view", "/Views/test/Index.cshtml", new object[]{ }),
+
+                        // This one doesn't have a RazorViewAttribute
```

Comment:
I don't follow this comment. Neither does `object` or `string`?

Reply:
This item doesn't have a RazorViewAttribute.